Automattic/wp-calypso
# Jetpack Connect: Move input redirections out of redux

The PR moves redirection actions which are only relevant for the site input step out of redux and into the component.

It also colocates some constants in a new constants file.

## Testing
1. Start with a new site with no Jetpack installed
1. Ensure installation works as before
1. Ensure activation works as before
1. Ensure auth works as before (moves to auth step correctly).
1. Ensure `goToPlans` works
   * enter a url for a site you already own on a free plan
   * You'll be redirected to `/jetpack/connect/plans` immediately
